scripts/ci/tags.yaml

```diff
@@ -612,3 +611,49 @@ sample_edge_impulse:
     - nrf/lib/wave_gen/
     - nrf/samples/edge_impulse/
     - zephyr/soc/nordic/
+
+ci_applications_nrf_desktop:
+  files:
+    - bootloader/mcuboot/
+    - modules/crypto/mbedtls/
+    - modules/crypto/oberon-psa-crypto/
+    - modules/crypto/tinycrypt/
+    - modules/lib/open-amp/
+    - modules/lib/zcbor/
+    - nrf/applications/ipc_radio/
+    - nrf/applications/nrf_desktop/
+    - nrf/boards/nordic/
+    - nrf/drivers/mpsl/temp_nrf5/
+    - nrf/drivers/sensor/paw3212/
+    - nrf/drivers/sensor/pmw3360/
+    - nrf/subsys/app_event_manager/
+    - nrf/subsys/bluetooth/
+    - nrf/subsys/bootloader/
+    - nrf/subsys/caf/
+    - nrf/subsys/fw_info/
+    - nrf/subsys/mgmt/
+    - nrf/subsys/mpsl/
+    - nrf/subsys/nrf_rpc/
+    - nrf/subsys/nrf_security/
+    - nrf/subsys/partition_manager/
+    - nrf/subsys/sdfw_services/
+    - nrf/subsys/suit/
+    - nrfxlib/crypto/
+    - nrfxlib/nrf_rpc/
+    - zephyr/drivers/usb/
+    - zephyr/boards/nordic/
```

Comment:
Please keep it ordered alphabetically

Reply:
Also `zephyr/cmake` and `zephyr/share`

Reply:
The same as for nrf/sysbuild and nrf/cmake
build system should not be verified by specific app, it should be checked by all of them.

Reply:
So maybe all apps should depend on it (to ensure we spot issues early)?